x509/common.go

```diff
@@ -65,3 +86,18 @@ func ValidateHsmArgs(hsmModule, hsmPin, hsmTokenLabel, moduleArg, pinArg, tokenA
 	}
 	return nil, nil
 }
+
+func parseOnePemBlock(pemBlock string) *pem.Block {
+	first, rest := pem.Decode([]byte(pemBlock))
+	if first == nil || len(rest) > 0 {
+		subcommands.DieNotNil(errors.New("Malformed PEM data"))
+	}
+	return first
+}
+
+func LoadCertFromFile(fn string) *x509.Certificate {
```

Comment:
Since this method is exported, could we add a comment on top to document it?

Reply:
I don't know anybody who is using Fioctl as a library.
So we have a privilege to avoid some excessive formalities.